recipe/meta.yaml

```diff
@@ -0,0 +1,161 @@
+{% set name = "torchaudio" %}
+{% set version = "2.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pytorch/audio/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 82c2745a73172b495327ec36c6970ad5ad9d5d2ac44feeaea2617152f9393bf7
+  patches:
+    - 0001-set-include-path.patch
+
+build:
+  number: 0
+  skip: True  # [py>311]
+  # temporarily skip windows until ffmpeg is available
+  script:
+    - export USE_FFMPEG=ON          # [not win]
+    - set USE_FFMPEG=ON             # [win]
+    - export BUILD_SOX=OFF          # [not win]
+    - set BUILD_SOX=OFF             # [win]
+    - export FFMPEG_ROOT=${PREFIX}  # [not win]
+    - set FFMPEG_ROOT=${PREFIX}     # [win]
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    - install_name_tool -add_rpath @loader_path/ $SP_DIR/torchaudio/lib/_torchaudio_ffmpeg.so  # [osx]
+    - install_name_tool -add_rpath @loader_path/ $SP_DIR/torchaudio/lib/_torchaudio.so         # [osx]
+    - patchelf --set-rpath '$ORIGIN' $SP_DIR/torchaudio/lib/_torchaudio_ffmpeg.so                  # [linux]
+    - patchelf --add-rpath '$ORIGIN/../../../..' $SP_DIR/torchaudio/lib/_torchaudio_ffmpeg.so      # [linux]
+    - patchelf --add-rpath '$ORIGIN/../../torch/lib' $SP_DIR/torchaudio/lib/_torchaudio_ffmpeg.so  # [linux]
+    - patchelf --set-rpath '$ORIGIN/../../torch/lib' $SP_DIR/torchaudio/lib/_torchaudio.so         # [linux]
+    - patchelf --add-rpath '$ORIGIN' $SP_DIR/torchaudio/lib/_torchaudio.so                         # [linux]
+  ignore_run_exports:
+    - pytorch-gpu  # [osx and arm64]
+    - libprotobuf  # [osx and arm64]
+    - libabseil    # [osx and arm64]
+  missing_dso_whitelist:
+    - lib/libstdc++.so.*            # [linux]
+    - lib/libgcc_s.so.*             # [linux]
+    - lib/libgomp.so.*              # [linux]
+    - $RPATH/ld64.so.1              # [linux and s390x]
+    - x86_64-conda-linux-gnu/sysroot/lib64/libm.so.*        # [linux and x64_64]
```

Comment:
likewise, loads of allowlist entries suggests something's up

Reply:
We can further analyze why this is. Some binaries seem to point to these libraries.